aporia/manifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1b4bbfb0-659e-4c67-b2a8-1d11853701b1",
+  "app_id": "aporia",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Cloud-Native ML Monitoring and Explainability",
+    "title": "Aporia",
+    "media": [
+      {
+        "caption": "Aporia Dashboard in Datadog",
+
+        "image_url": "images/aporia-datadog-dashboard.png",

```

Comment:
Please retake a screenshot of this dashboard that contains the Aporia logo, and detailed information populated in the dashboard. 

The image is currently crunched in size with no logo, and contains the "Add Widgets" box which indicates the dashboard is incomplete or at least in edit mode.